downstream/modules/platform/proc-create-keycloak-client.adoc

```diff
@@ -0,0 +1,143 @@
+[id="proc-create-keycloak-client_{context}"]
+
+= Creating a client
+
+Use the following procedure to create the Client.
```

Comment:
Same comment as before. For an abstract I would expect one, or ideally two, sentences that describe the user motivation. What does the Keycloak client do? Why should you create one?